packages/features/insights/components/RoutingFormResponsesTable.tsx

```diff
@@ -345,6 +366,10 @@ export function RoutingFormResponsesTable() {
     },
   });
 
+  useEffect(() => {
+    onTableReady?.(table);
+  }, [table, onTableReady]);
```

Comment:
When talbe loads we can pass table to filter bar that lives outside of the data-table. In future we can move this inside and use the DataTableToolbar component but its too big of a refactor